docs/get-started/developers/installation.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: layout-with-subnav.njk
+title: Installation
+heading: Developers
+tags:
+  - developers
+permalink: /get-started/developers/installation/index.html
+order: 10
+bodyClasses: element-docs
+---
+
+## How to install
```

Comment:
Brian commented:

With this section, should we instead link over to the nvm repo - [https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating](https://www.google.com/url?q=https://github.com/nvm-sh/nvm?tab%3Dreadme-ov-file%23installing-and-updating&sa=D&source=docs&ust=1705530502157537&usg=AOvVaw2szLgCsrZHdZGQrCQ6EE4M) instead of maintaining our own docs for installing and using nvm?

Reply:
> With this section, should we instead link over to the nvm repo - [https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating](https://www.google.com/url?q=https://github.com/nvm-sh/nvm?tab%3Dreadme-ov-file%23installing-and-updating&sa=D&source=docs&ust=1705530502157537&usg=AOvVaw2szLgCsrZHdZGQrCQ6EE4M) instead of maintaining our own docs for installing and using nvm?

Our docs will fall out of date if we try and own that content, its best to send them to the website and only maintain the link.